k8s/ingress.yml

```
@@ -2,28 +2,31 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: chatapp-ingress
-  namespace: chat-app
+  namespace: chatapp
+  annotations:
   labels:
     name: chatapp-ingress
   annotations:
```

YAMLlint (1.37.1)

[error] duplication of key "annotations" in mapping

(key-duplicates)